torch/export/experimental/__init__.py

```diff
@@ -420,13 +420,15 @@ def _compiled_and_package(
                     path = Path(base_directory) / f"{name}_input_{i}.pt"
                     torch.save(t, path)
 
-        cmake_file_str = _get_make_file(package_name, model_names, use_cuda)
+        # Detect if ROCm is being used
+        is_hip = bool(getattr(torch.version, "hip", None))
```

Comment:
Why not just `torch.version.hip`?  Or `torch.version.hip is not None`?  Are you finding cases where `hip` as an attr doesn't exist?